cfgov/jinja2/v1/about-us/careers/students-and-graduates/index.html

```diff
@@ -73,8 +69,14 @@ <h3 class="h5">
         </div>
         <div class="media_body">
             <h2>CFPB Summer Internship</h2>
-            <h3 class="h5">Look for applications here starting Fall 2016</h3>
-            <p>
+            <div class="h5">
+                Applications open 10/20/16
+            </div>
+            <a class="jump-link jump-link__download"
+               href="http://files.consumerfinance.gov/f/documents/cfpb_RecruitmentFlyer_SummerInternship.pdf">
```

Comment:
We didn't; we're moving away from it. This is one of the last vestiges. Just not quite ready for it to go into Wagtail, and this update is urgent.